java/vector/src/test/java/org/apache/arrow/vector/TestRunEndEncodedVector.java

```diff
@@ -78,6 +79,18 @@ public void testConstantValueVector() {
       for (int i = 0; i < logicalValueCount; i++) {
         assertEquals(value, reeVector.getObject(i));
       }
+
```

Comment:
nit: I would rather not mix tests focused on evaluating particular nature of the vector with another one.